addons/l10n_es_edi_facturae/tests/test_edi_xml.py

```diff
@@ -90,6 +90,15 @@ def create_invoice(cls, **kwargs):
             ],
         })
 
+    def create_send_and_print(self, invoices, **kwargs):
+        template = self.env.ref(invoices._get_mail_template())
+        return self.env['account.move.send']\
+            .with_context(active_model='account.move', active_ids=invoices.ids)\
+            .create({
+                'mail_template_id': template.id,
+                **kwargs,
+            })
```

Comment:
```suggestion
        return self.env['account.move.send'].with_context(
            active_model='account.move',
            active_ids=invoices.ids,
        ).create({
            'mail_template_id': template.id,
            **kwargs,
        })
```

https://peps.python.org/pep-0008/#maximum-line-length
> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.